x-pack/plugins/code/public/components/admin_page/admin.tsx

```diff
@@ -49,6 +49,19 @@ interface State {
 }
 
 class AdminPage extends React.PureComponent<Props, State> {
+  public static getDerivedStateFromProps(props: Props) {
```

Comment:
where is this function used?

Reply:
this overrides a react component lifecycle function